modules/services/kbfs.nix

```diff
@@ -33,34 +33,74 @@ in
           Additional flags to pass to the Keybase filesystem on launch.
         '';
       };
+
+      fusermountPath = mkOption {
+        type = types.str;
+        default = "/run/wrappers/bin";
+        example = "/usr/bin";
+        description = ''
+          Path to the directory containing the <citerefentry>
```

Comment:
Perhaps good to clarify that it should be an _absolute_ path? Also, perhaps it would be good to make this point directly to the `fusermount` executable, not the directory holding it? The option is called `fusermountPath` and not `directoryContainingFusermountPath`, after all 🙂